docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -5,6 +5,19 @@
 This document outlines best practices for Aerogear APB development.
 General information about the OpenShift Ansible Broker (OAB) and Ansible Playbook Bundles (APB) is included in the link:https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/ansible_service_broker.html[OpenShift docs].
 
+== Prerequisites
+
+Before doing local development of Aerogear APBs, you should have used the installer in this repo to set up a local OpenShift cluster. link:https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/walkthroughs/local-setup.adoc[Read the document here.]
+
+Additionally, the `developer` user in OpenShift needs some extra permissions. Use the `system:admin` user to apply the permissions and then log back in as `developer`.
+
+```bash
+oc login -u system:admin
```

Comment:
Really cool to have. Maybe we can add that to installer at some point.

Reply:
Absolutely. I'm not sure whether the installer is specifically geared towards local dev or production installs but it might be nice to have a flag of some sort that would run these commands

Reply:
Installer is currently geared towards development, but could be the inspiration or starting point for a production (or at least remote cluster) install.

I'm OK with the developer user having the extra perms added during install.
However, part of me has been hesitant as it may lead us to inadvertently add functionality that requires admin privileges without realising it.
I'd feel better suggesting this if @maleck13 was on board with it too

Reply:
generally I prefer to create a different user.
```
oc login
admin 
admin

oc login -u system:admin
oc adm policy add-cluster-role-to-user admin admin

oc login
admin 
admin
````
This gives 3 distinct users
- developer who logins in with "regular" perms
- admin who can login to the UI with cluster admin perms
- system:admin who can't login to UI but is also an admin.

Reply:
Thanks for your input @maleck13. The approach I've outlined here is the one that organically formed on our team (as a result of having no clear documentation at the time). It will be good to put something 'more official' in place.